packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -4,13 +4,7 @@
 
 'use strict';
 
-const PRODUCT_SUBSCRIBED = 'defaultSubscribed';
```

Comment:
Removing this default subscribed / registered feature from capabilities.

We had originally implemented it to support SUMO for detection of whether a user was subscribed to anything at all. But, in practice, I think [we just added the SUMO client ID to each product as a capability](https://github.com/mozilla-services/cloudops-deployment/blob/84b9cd8612dc09a2b92ef375c197c6b0517f0892/projects/fxa/puppet/yaml/type/fxa.auth_server.prod.yaml#L70) and we should ensure that's carried over to Stripe metadata.